### PR TITLE
Features/daniv project tree view

### PR DESCRIPTION
**Pull request by DANIV18**

### CODE REVIEW

**Review by vsakharevych — APPROVED** [review]
No remarks.

**Review by valentynfaychuk — APPROVED** [review]
Approving directory tree

File: src/projectviewermodel/projectviewermodel.cpp [hunk 1]
```diff
@@ -29,18 +29,16 @@ ProjectViewerModel::~ProjectViewerModel()
 
 QVariant ProjectViewerModel::data(const QModelIndex &index, int role) const
 {
+
     if(!index.isValid())
     {
         return QVariant();
     }
+    QFileInfo info = fileInfo(index);
```
Comment (FedorenkoAndriy) [inline src/projectviewermodel/projectviewermodel.cpp:37]: You don't use this variable

File: src/projectviewermodel/projectviewermodel.cpp [hunk 2]
```diff
@@ -29,18 +29,16 @@ ProjectViewerModel::~ProjectViewerModel()
...
 
     switch (role)
     {
-    case Qt::DisplayRole:
-    {
-        QFileInfo info = fileInfo(index);
-        return info.fileName();
-    }
     case Qt::DecorationRole:
         if(isDir(index))
```
Comment (FedorenkoAndriy) [inline src/projectviewermodel/projectviewermodel.cpp:43]: You can use operator : ?

**Pull request merged**